src/docs/patterns/overlay.mdx

```diff
@@ -1,73 +1,190 @@
+They can also present simple form data and inputs, such as radio buttons, checkbox, dropdown menus, etc. for editing a selected item. This type of sidebar should be dismissible, such that the user can easily close it with a close button or clicking elsewhere on the screen.
 
-## Bottom Sheet
+Sidebars should always appear on the opposite side of the screen as the main [navigation](/patterns/navigation).
+
+> In general, modals should be used when creating new data, and sidebars should be used for viewing and editing existing data.
 
-A bottom sheet is similar to a modal dialog but slides in from the bottom of the screen. A dark overlay is usually used to block user interaction with the main app until the bottom sheet is closed.
+## Bottom Sheet
 
 <MaterialDesignDescription title={'Sheets: Bottom'} url={'https://material.io/components/sheets-bottom/'} />
 
-This is a very common pattern in mobile applications but can also be used in responsive web apps. Typically, the bottom sheet contains global actions for the entire page and is activated by a button in the app bar.
-
-<ImageGrid images={[BasicBottomsheet, ComplexBottomsheet]} caption={'A Bottom Sheet can be simple or complex.'} />
-
-### Basic
-
-<DemoCard
-    repository={'bottomsheet'}
-    angular={
-        'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Foverlays%2Fbasic-bottom-sheet%2Fbasic-bottom-sheet.component.ts'
-    }
-    react={
-        'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/overlays/basic-bottom-sheet/index.tsx&initialpath=basic-bottom-sheet'
-    }
-    ionic
-    reactNative
-/>
-
-The basic bottom sheet simply contains a list of actions. The final "Cancel" action is used to dismiss the bottom sheet.
-
-### Advanced
-
-<DemoCard
-    repository={'complex-bottomsheet'}
-    angular={
-        'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Foverlays%2Fcomplex-bottom-sheet%2Fcomplex-bottom-sheet.component.ts'
-    }
-    react={
-        'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/overlays/complex-bottom-sheet/index.tsx&initialpath=complex-bottom-sheet'
-    }
-    ionic
-    reactNative
-/>
-
-A bottom sheet is essentially just a container with some attached behaviors. Though it typically contains a simple list, it can be customized to include any content that is required for your application. For example, it can be used for custom sort/filter functionality (seen in the example above), biometric authentication prompts, etc.
+Bottom sheets are content surfaces that slide in from the bottom of the screen, primarily used in mobile applications. There are three types (standard, modal, and expandable), but the one used most often in PX Blue applications is the modal bottom sheet.
```

Comment:
abuse of brackets :P

Reply:
> primarily used in mobile applications

Would you say that on mobile it _always_ triggers a bottomsheet, and on desktop it _always_ triggers a contextual menu?

Reply:
I wouldn't say always. I'm sure there is a use case for Bottomsheet on desktop. Mobile is probably always going to be a bottomsheet, but I can't say that with 100% confidence right now.

Reply:
yeah I am asking bc I played around with an android phone (gmail client in particular). It seems that for the same list item interaction, android gmail is doing menu, while iOS is doing bottomsheet.

Reply:
🤷‍♂️  nobody can do anything the same